DailyCodingProblem/Program.cs

```diff
@@ -42,5 +44,51 @@ static void Problem1()
 
             Console.WriteLine($"Sorry! There is not two numbers in the list that add up to {number}");
         }
+
+
+        /// <summary>
+        /// Given an array of integers, return a new array such that each element of the new array is the product of all the numbers in the original array except the one at the current index.
+        /// </summary>
+        static void Problem2()
+        {
+            Console.WriteLine("I need you to give me at least five integers. Please enter one at the time!");
+            List<int> listOfNumbers = new List<int>();
+
+            while(listOfNumbers.Count < 5)
```

Comment:
Your instructions imply they can enter more than 5 numbers, but I don't see how that is possible here...

So I would either change the instructions or allow more than 5.